projects/cn0561/src/cn0561.c

```diff
@@ -65,6 +65,8 @@
 #include "clk_axi_clkgen.h"
 #include "axi_dmac.h"
 
+#define CN0561_ZED_CARRIER
```

Comment:
please remove. this is handled in the builds.json. Looks like you added it only for testing locally.